component/admin/models/media.php

```diff
@@ -50,6 +50,14 @@ public function _buildQuery()
         if ($media_type = $this->getState('media_type')) {
             $query->where('media_type = ' . $db->q($media_type));
         }
+        
+        $filter_search = $this->getState('filter_search', null);
+        if ($filter_search) {
```

Comment:
line break after line 54 please!

Reply:
Please help me check again, I don't know why your comment not disappear